dotnet/src/SemanticKernel.Abstractions/Functions/KernelFunction.cs

```diff
@@ -509,7 +509,6 @@ private void LogFunctionResult(ILogger logger, string? pluginName, string functi
     /// The <see cref="Kernel"/> instance to pass to the <see cref="KernelFunction"/> when it's invoked as part of the <see cref="AIFunction"/>'s invocation.
     /// </param>
     /// <returns>An instance of <see cref="AIFunction"/> that, when invoked, will in turn invoke the current <see cref="KernelFunction"/>.</returns>
-    [Experimental("SKEXP0001")]
```

Comment:
Not fully sure about this one, I would keep any MEAI conversion for a bit still. We might be able to have KernelFunctin : AiFunction (haven't explore it yet) but if that happens and we do have native support this API won't be necessary anymore.